stylus/components/empty.styl

```diff
@@ -1,6 +1,9 @@
 @require '../settings/palette'
 @require '../settings/breakpoints'
 @require '../tools/mixins'
+@require '../objects/layouts'
+
+contentHeight = barHeight * 2
```

Comment:
IMHO It is a bit unclear that it comes from the nav and the bar heights combined (and it is a bit of a coincidence that the two bars have the same height). I think it would be clearer to have `contentHeight = barHeight + navHeight` and have navHeight also defined in `objects/layouts`.

Reply:
It's not a coincidence per se, but I get your point, in case things change, even in other context, it'll be ready. Gonna change that.